hudi-common/src/main/java/org/apache/hudi/common/config/HoodieStorageConfig.java

```diff
@@ -52,6 +52,16 @@ public class HoodieStorageConfig extends HoodieConfig {
       .withDocumentation("Parquet page size in bytes. Page is the unit of read within a parquet file. "
           + "Within a block, pages are compressed separately.");
 
+  public static final ConfigProperty<Long> PARQUET_MIN_ROW_COUNT_FOR_SIZE_CHECK = ConfigProperty
```

Comment:
I don't think we should be exposing this configs to the user:
 - These are extremely low-level and 
 - Instead we should just set sensible defaults for these

On top of that nobody should actually be controlling these

Reply:
sure, let me remove that